Error message when using zeromq.js in Electron on Win7 system

message：index-4ba7dd8b.js:13 Error: The specified module could not be found.
\\?\D:\Program Files\platform_client\resources\app\node_modules\zeromq\prebuilds\win32-x64\node.napi.glibc.node
    at process.func [as dlopen] (node:electron/js2c/asar_bundle:5:1810)
    at Module._extensions..node (node:internal/modules/cjs/loader:1205:18)
    at Object.func [as .node] (node:electron/js2c/asar_bundle:5:1810)
    at Module.load (node:internal/modules/cjs/loader:988:32)
    at Module._load (node:internal/modules/cjs/loader:829:12)
    at c._load (node:electron/js2c/asar_bundle:5:13343)
    at i._load (node:electron/js2c/renderer_init:33:356)
    at Module.require (node:internal/modules/cjs/loader:1012:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at load (D:\Program Files\platform_client\resources\app\node_modules\@aminya\node-gyp-build\node-gyp-build.js:22:10)

--- a/.github/workflows/CI.yml
+++ b/.github/workflows/CI.yml
@@ -13,9 +13,8 @@ jobs:
       fail-fast: false
       matrix:
         os:
-          # - ubuntu-22.04
           - ubuntu-20.04
-          - windows-2022
+          - windows-2019
         node_version:
           - 18
         node_arch:
@@ -28,7 +27,7 @@ jobs:
           - ""
 
         include:
-          - os: windows-2022
+          - os: windows-2019
             node_version: 18
             node_arch: ia32
             cpp_arch: amd64_x86

